python/README-building-wheels.md

```diff
@@ -9,7 +9,7 @@ produce .whl files owned by root.
 Inside the root directory (i.e., one level above this python directory), run
 
 ```
-docker run --rm -w /ray -v `pwd`:/ray -ti rayproject/arrow_linux_x86_64_base:latest /ray/python/build-wheel-manylinux1.sh
+docker run -e TRAVIS_COMMIT=<commit_number_to_use> --rm -w /ray -v `pwd`:/ray -ti rayproject/arrow_linux_x86_64_base:python-3.8.0 /ray/python/build-wheel-manylinux1.sh
```

Comment:
I'm not sure why we need `TRAVIS_COMMIT` here, but the `python-3.8.0` change looks good to me!

Reply:
It's needed because [`build-wheel-manylinux1.sh`](https://github.com/ray-project/ray/blob/6c49c01837ef9d83b998a2ec1a269a1a6767fc66/python/build-wheel-manylinux1.sh#L62) requires it: